data/com.endlessm.GoogleChromeInitialSetup.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=Google Chrome Initial Setup
+Exec=/usr/bin/eos-google-chrome-installer
```

Comment:
Oh yes, sorry for the confusion. I meant the libexec dir, not the arch dependant one.